Include/internal/_mem.h

```diff
@@ -11,6 +11,9 @@ extern "C" {
 #include "_pymalloc.h"
 #endif
 
+#undef  uint
+#define uint    unsigned int    /* assuming >= 16 bits */
```

Comment:
Oh come on, this is lazy :P It's okay for testing, but I'll insist you change all uses of `uint` into a standard type name before merging.